README.md

```diff
@@ -1 +1,53 @@
-# ontology
\ No newline at end of file
+![Industrial Ontologies Foundry](https://i0.wp.com/industrialontologies.org/wp-content/uploads/2020/01/cropped-IOF-LOGO-2-4.png)
+
+# Industrial Ontologies Foundry
+
+The IOF (The Industrial Ontologies Foundry) is a unit of [OAGi](https://OAGi.org) (Open Applications Group), a non-profit organization with the mission to reduce the cost of integration by developing inter-operable, cross-functional, cross-industry, data-model-driven, and extensible standards to meet the challenge of a rapidly-changing global digital economy. 
+
+IOF's mission is to create a suite of ontologies intended to support digital manufacturing by facilitating cross-system integration within the factory and across an enterprise; in commerce between suppliers, manufacturers, customers, and other trading partners; and throughout the various stages of the product life cycle. The IOF ontologies consist of a common mid-level ontology, "Core," and several domain specific ontologies.
+
+The IOF Core Ontology resides at the top of this suite from an architectural perspective and contains terms found in several operational areas of manufacturing. Additionally, the architectural approach chosen by the IOF is to base all of its ontologies on a single foundational or top-level ontology–for which the IOF chose the [Basic Formal Ontology](https://basic-formal-ontology.org/bfo-2020.html) or BFO. The Core Ontology contains many intermediate-level terms that derive from BFO and from which the IOF ontologies derive domain industry terms. Core intermediate-level terms are often domain independent–meaning one can find them in other industries and fields, such as in the banking, insurance, and healthcare industries, or the sciences, as in the physics, chemistry, and biology domains. 
+
```

Comment:
```suggestion
# Definition of Status
Following is the detail definitions and usage guide of the Maturity Level (excerpt from [IOF Annotation Vocabulary Guide v2.1](https://oagiscore.atlassian.net/wiki/spaces/IOF/pages/4532142081/IOF+Annotation+Property+Guide+V2.1#Maturity). Note that access to the guide may require [IOF membership](https://industrialontologies.org/participation-request/))

* **maturity** – `iof-av:maturity`* 
 
  * Definition: annotation property used to indicate the development status of a resource such as an ontology or a construct

  * Each IOF ontology **MUST** have exactly one maturity annotation with a value of type iof-av:MaturityLevel

  * Each construct within an ontology will default to the (i.e., be assumed to have the same) maturity of the ontology unless an explicit maturity annotation for the construct indicates otherwise

  * Each construct **MAY** have a maturity annotation, indicating the specific construct’s status of development

  * One **MUST** use the following vocabulary when specifying the maturity:

      * iof-av:Released 

        * Will not be removed from the ontology for a reasonable length of time

        * Indicates an ontology or construct that is considered to be stable and mature from a development perspective

        * Release notes will be provided for any changes concerning released content, and any revisions will be backward compatible with the prior version to the degree possible

      * iof-av:Provisional 

        * Indicates an ontology or construct that is considered to be under development

        * Provisional content is subject to change and may change substantially before release. IOF users should be aware that it is not dependable but could be used for reference and as the basis for further work

```

Reply:
Will check how this renders